### PR TITLE
Add localization messages for 'sv' locale

### PR DESCRIPTION
**Pull request by BingoRUS**
Swedish localization messages are provided by @aenges.

### CODE REVIEW

File: js/localization/messages/sv.json [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+        "dxDataGrid-ungroupHeaderText": "Avgruppera",
+        "dxDataGrid-ungroupAllText": "Avgruppera allt",
+        "dxDataGrid-editingEditRow": "Redigera",
+        "dxDataGrid-editingSaveRowChanges": "Spara",
+        "dxDataGrid-editingCancelRowChanges": "Avbryt",
+        "dxDataGrid-editingDeleteRow": "Radera",
+        "dxDataGrid-editingUndeleteRow": "Ångra radering",
+        "dxDataGrid-editingConfirmDeleteMessage": "Är du säker på att du vill radera denna post?",
+        "dxDataGrid-validationCancelChanges": "Avbryt ändringarna",
+        "dxDataGrid-groupPanelEmptyText": "Dra en kolumnrubrik hit för att dölja den",
```
Comment (BingoRUS) [inline js/localization/messages/sv.json]: @aenges It's the constant about group panel, not about column chooser. Dragging a column there will group DataGrid by this column. How could we change this constant?

File: js/localization/messages/sv.json [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+        "dxDataGrid-headerFilterEmptyValue": "(Tomma)",
+        "dxDataGrid-headerFilterOK": "OK",
+        "dxDataGrid-headerFilterCancel": "Avbryt",
+        "dxDataGrid-ariaColumn": "Kolumn",
+        "dxDataGrid-ariaValue": "Värde",
+        "dxDataGrid-ariaFilterCell": "Filtrera cell",
+        "dxDataGrid-ariaCollapse": "Kollapsa",
+        "dxDataGrid-ariaExpand": "Expandera",
+        "dxDataGrid-ariaDataGrid": "Datarutnät",
+        "dxDataGrid-ariaSearchInGrid": "Säk i datarutnätet",
```
Comment (BingoRUS) [inline js/localization/messages/sv.json]: @aenges Is it the right translation for "search" or it should be "Sök"?

Comment (aenges) [reply js/localization/messages/sv.json]: Sorry, should be "Sök".

File: js/localization/messages/sv.json (hunk 1 above)
Comment (aenges) [reply js/localization/messages/sv.json]: "dxDataGrid-groupPanelEmptyText": "Dra en kolumnrubrik hit för att gruppera enligt den kolumnen",

**Review by San4es — APPROVED** [review]

**Pull request merged**